### DISCUSSION TITLE
Flaky Test Tracker

### DISCUSSION THREAD

**Discussion by potatowagon** [facebook/lexical#6289] [General]
**If a flaky test caused failure on a workflow run, please record it here.**

How to use:
1. Find (ctrl + f) if the name of the flaky test is already recorded here. If no, add it as a comment on this thread.
2. Reply to the comment the Github Workflows name + link to the flaky test output. Just select the workflow title, copy and paste in comments:
<img width="463" alt="Screenshot 2024-06-10 at 5 37 26 PM" src="https://github.com/facebook/lexical/assets/19604232/13f686d6-ac29-411a-9662-2a4d98edd6a2">

3. Please do not comment anything else aside from test name and workflows link.

What happens:
1. Lexical team will periodically raise a PR tagging these tests as flaky.
2. Tests tagged flaky will be ran in a separate CI (wip).
3. You can use this tracker to reference if the CI is failing due to a flaky test.

**Comment by potatowagon** [comment]
Toolbar.spec.mjs:37:3 › Toolbar › Insert image caption + table

**Reply by potatowagon** [reply]
[e2e-tests / collab-windows (18.18.0, firefox) / e2e-test](https://github.com/facebook/lexical/actions/runs/9409582384/job/25919783165#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6365

**Comment by potatowagon** [comment]
ClearFormatting.spec.mjs:109:3 › Clear All Formatting › Should preserve the default styling of hashtags and mentions

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, webkit, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9444245507/job/26009440131?pr=6284#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6365

**Comment by potatowagon** [comment]
Links.spec.mjs:2003:3 › Links › Can handle pressing Enter at the beginning of a Link

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, chromium) / e2e-test](https://github.com/facebook/lexical/actions/runs/9371922530/job/25802279025#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6365

**Comment by potatowagon** [comment]
History.spec.mjs:29:3 › History › Can type two paragraphs of text and correctly undo and redo

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, webkit, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9438959246/job/25996513412#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6365

**Comment by potatowagon** [comment]
List.spec.mjs:1434:3 › Nested List › remove list breaks when selection in empty nested list item 2

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, chromium) / e2e-test](https://github.com/facebook/lexical/actions/runs/9460109086/job/26058340370#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6365

**Comment by potatowagon** [comment]
Tables.spec.mjs:2462:3 › Tables › Delete rows (with conflicting merged cell)

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9850595424/job/27196066505#logs)

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9802028396/job/27066148674#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6388

**Comment by potatowagon** [comment]
4697-repeated-table-selection.spec.mjs:21:3 › Regression test #4697 › repeated table selection results in table selection

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9849749557/job/27193939392#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6388

**Comment by potatowagon** [comment]
Tables.spec.mjs:1614:3 › Tables › Resize merged cells width (2)

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, firefox) / e2e-test](https://github.com/facebook/lexical/actions/runs/9849749557/job/27193934778#logs)

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, firefox) / e2e-test](https://github.com/facebook/lexical/actions/runs/9849749557/job/27193934778#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6388

**Comment by potatowagon** [comment]
Tables.spec.mjs:2976:3 › Tables › Can align text using Table selection

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, firefox) / e2e-test](https://github.com/facebook/lexical/actions/runs/9789342234/job/27028908164#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6388

**Comment by potatowagon** [comment]
Tables.spec.mjs:1051:3 › Tables › Can clear text using Table selection

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, firefox) / e2e-test](https://github.com/facebook/lexical/actions/runs/9864687570/job/27240133879#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6405/

**Comment by potatowagon** [comment]
4872-full-row-span-cell-merge.spec.mjs:21:3 › Regression test #4872 › merging two full rows does not break table selection

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9864687570/job/27240140066#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6405/

**Comment by potatowagon** [comment]
Tables.spec.mjs:969:3 › Tables › Can copy + paste (internal) using Table selection

**Reply by potatowagon** [reply]
[e2e-tests / mac (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9895555346/job/27335756381#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6405/

**Comment by potatowagon** [comment]
Tables.spec.mjs:878:3 › Tables › Can style text using Table selection

**Reply by potatowagon** [reply]
[e2e-tests / linux (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/9968244326/job/27543187880?pr=6397#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6462

**Comment by potatowagon** [comment]
List.spec.mjs:1307:3 › Nested List › can navigate and check/uncheck with keyboard

**Reply by potatowagon** [reply]
[e2e-tests / collab-prod (macos-latest, 18.18.0, chromium, rich-text-with-collab, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10095528967/job/27915940500?pr=6460#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6462

**Comment by potatowagon** [comment]
Selection.spec.mjs:483:3 › Selection › Can delete sibling elements forward

**Reply by potatowagon** [reply]
[e2e-tests / linux (18.18.0, chromium, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10053265014/job/27785760882#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6462

**Comment by potatowagon** [comment]
Tables.spec.mjs:2258:3 › Tables › Merge multiple merged cells and then unmerge

**Reply by potatowagon** [reply]
[e2e-tests / collab-linux (18.18.0, firefox) / e2e-test](https://github.com/facebook/lexical/actions/runs/10794893688/job/29940233856#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6620

**Comment by potatowagon** [comment]
Tables.spec.mjs:1309:3 › Tables › Grid selection: can select multiple cells and insert an image

**Reply by potatowagon** [reply]
[e2e-tests / collab-prod (ubuntu-latest, 18.18.0, chromium, rich-text-with-collab, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10793011847/job/29933971908#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6620

**Comment by potatowagon** [comment]
Tables.spec.mjs:2129:3 › Tables › Select multiple merged cells (selection expands to a rectangle)

**Reply by potatowagon** [reply]
[e2e-tests / mac-rich (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10793999963/job/29937308040#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6620

**Comment by potatowagon** [comment]
Tables.spec.mjs:2627:3 › Tables › Delete columns backward

**Reply by potatowagon** [reply]
[e2e-tests / mac-rich (18.18.0, firefox, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10733648641/job/29767382268#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6620

**Comment by potatowagon** [comment]
Links.spec.mjs:429:3 › Links › Can create a link with some text after, insert paragraph, then backspace, it should merge correctly

**Reply by potatowagon** [reply]
[e2e-tests / collab-prod (ubuntu-latest, 18.18.0, chromium, rich-text-with-collab, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10596566884/job/29364824024#logs)

**Reply by potatowagon** [reply]
marked flaky in https://github.com/facebook/lexical/pull/6620

**Comment by etrepum** [comment]
```
[firefox] › packages/lexical-playground/__tests__/e2e/Tables.spec.mjs:2701:3 › Tables › Delete columns (with conflicting merged cell) 
```
Failed in #6619 [e2e-tests / collab-prod (ubuntu-latest, 18.18.0, chromium, rich-text-with-collab, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/10596566884/job/29364824024#logs)

**Comment by etrepum** [comment]
```
    [chromium] › packages/lexical-playground/__tests__/e2e/List.spec.mjs:75:3 › Nested List › Can create a list and partially copy some content out of it 
```

Failed in https://github.com/facebook/lexical/pull/6932 [e2e-tests / canary (ubuntu-latest, 18.18.0, chromium, rich-text, modern-events) / e2e-test](https://github.com/facebook/lexical/actions/runs/12249491935/job/34170887424?pr=6932#logs)
